### DISCUSSION TITLE
[MVVM] How to bind a collection to a StackPanel instead of a ListBox?

### DISCUSSION THREAD

**Discussion by slxdy** [AvaloniaUI/Avalonia#15967] [Q&A] [answered]
It's hard to explain this in words, so let me show an example.

This is what I currently have:
```xaml
<ListBox ItemsSource="{Binding Games}" Background="Transparent">
    <ListBox.ItemTemplate>
        <DataTemplate>
            <views:GameControl DataContext="{Binding}" />
        </DataTemplate>
    </ListBox.ItemTemplate>
</ListBox>
```

And I'd like to turn it into something like:

```xaml
<StackPanel ItemsSource="{Binding Games}" Orientation="Vertical">
    <StackPanel.ItemTemplate>
        <DataTemplate>
            <views:GameControl DataContext="{Binding}" />
        </DataTemplate>
    </StackPanel.ItemTemplate>
</StackPanel>
```

How do I achieve that?

**Comment by stevemonaco** [comment, marked as answer]
`StackPanel` isn't a collection control, it's a layout control. I'm guessing that you don't want selection in which case you should use `ItemsControl` instead. You can replace the `ItemsPanel` of a collection control, but `ItemsControl` already has a `StackPanel` as its default.

**Reply by slxdy** [reply]
Perfect! Thanks
